repose-aggregator/artifacts/cli-utils/build.gradle

```diff
@@ -3,6 +3,8 @@ apply plugin: 'nebula.ospackage'
 
 configurations {
     fatJar
+    deb
```

Comment:
Forgot to clean up with the others?

Reply:
Yeah. Well, forgot to add these configurations to this project when I did the others... so it's here now.